ONSdigital/design-system
# Collapsible modules and components refactor

How to test:
- Run 'yarn start' to check behaviour from user level
- Run 'yarn test' to check functional tests
